File: 800/2116A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int a, b, c, d;
        cin >> a >> b >> c >> d;

        int mini = min({a, b, c, d});

        if (mini == b || mini == d)
            cout << "Gellyfish\n";
        else
            cout << "Flower\n";
    }

    return 0;
}

File: 800/2116A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while(t-- > 0)
        {
            int a = sc.nextInt();
            int b = sc.nextInt();
            int c = sc.nextInt();
            int d = sc.nextInt();

            int mini = Math.min(a, b);
            mini = Math.min(c, mini);
            mini = Math.min(d, mini);

            if(mini == b || mini == d)
                System.out.println("Gellyfish");
            else
                System.out.println("Flower");
        }
    }
}